HIP/hip-904-frictionless-airdrop.md

```diff
@@ -752,25 +815,35 @@ was transferred.
 
 ```protobuf
 /**
- * Reject undesired tokens.<br/>
+ * Reject undesired token(s).<br/>
  * Transfer one or more token balances held by the requesting account to the treasury for each
  * token type.<br/>
  * Each transfer SHALL be one of the following
  * - A single non-fungible/unique token.
  * - The full balance held for a fungible/common token type.
- * 
+ *
  * A single tokenReject transaction SHALL support a maximum of 10 transfers.
  */
 message TokenRejectTransactionBody {
-    /**
-     * A list of zero of more fungible/common token types.
-     */
-    repeated TokenID fungible_token_ids = 1;
+    repeated TokenRejection rejections = 1;
```

Comment:
By introducing a message and repeating that here, we can clearly limit each transaction to no more than 10 items in total.